docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -566,6 +566,33 @@ Here one could find description of general HTTP properties applied across all th
 
 |===
 
+== Properties externalization
+
+. Declare https://en.wikipedia.org/wiki/Environment_variable[environment variable] containing desired property value and then 
+point the property to the declared environment variable:
++
+[source,properties]
+----
+bdd.variables.global.input-text=${MY_INPUT_TEXT}
+----
++
+[source,shell]
+----
+export MY_INPUT_TEXT=kekw
+./gradlew runStories
+----
+
+. Pass the property with its value into a test run using command line arguments prepending it with the `-Pvividus.` prefix. 
+The prefix only plays role during test start-up phase and cut off during the actual test run:
++
+[source,shell]
+----
+./gradlew runStories -Pvividus.bdd.variables.global.input-text=value1 -Pvividus.bdd.variables.global.output-text=value2
+----
+
+. Generate the `overriding.properties` file on-the-fly and put it into corresponding folder in a test project.
```

Comment:
I think we need some example here as well

Reply:
should I add for both windows and linux platforms?

Reply:
linux should be enough